sdk/helper/ldaputil/ldap.go

```diff
@@ -13,16 +15,19 @@ func NewLDAP() LDAP {
 // LDAP provides ldap functionality, but through an interface
 // rather than statically. This allows faking it for tests.
 type LDAP interface {
-	Dial(network, addr string) (Connection, error)
-	DialTLS(network, addr string, config *tls.Config) (Connection, error)
+	Dial(addr string) (Connection, error)
```

Comment:
The changes to the interface are probably not needed and potentially could cause issues outside of the project (OpenLDAP and AD engines come to mind).  We can just ignore `network` and use `DialURL` instead of `Dial`.